src/task.ts

```diff
@@ -126,6 +130,33 @@ export async function addProvingTask(
     private_inputs: priv_inputs,
     proof_submit_mode: proof_submit_mode,
   };
+
+  if (context_file) {
+    let contextBytes: Buffer = fs.readFileSync(context_file);
+    if (input_context_type !== InputContextType.Custom) {
```

Comment:
This checking part better before the "let contextBytes: Buffer = fs.readFileSync(context_file);"?